subprojects/ide-native/src/integTest/groovy/org/gradle/ide/xcode/XcodeSingleCppProjectIntegrationTest.groovy

```diff
@@ -202,10 +204,11 @@ apply plugin: 'cpp-application'
         then:
         resultDebug.assertTasksExecuted(':dependDebugCpp', ':compileDebugCpp', ':linkDebug', ':_xcode___App_Debug')
         resultDebug.assertTasksNotSkipped(':dependDebugCpp', ':compileDebugCpp', ':linkDebug', ':_xcode___App_Debug')
-        exe("build/exe/main/debug/App").exec().out == app.expectedOutput
+        debugBinary.exec().out == app.expectedOutput
+        fixture(debugBinary).assertHasDebugSymbolsFor(app.sourceFileNamesWithoutHeaders)
```

Comment:
I was actually thinking of something more like `debugBinary.assertHasDebugSymbolsFor()` like you've got in later tests.